### DISCUSSION TITLE
Android and iOS repos private

### DISCUSSION THREAD

**Discussion by Randomneo** [stoatchat/discussions#674] [General]
Why Android and iOS client app repos are private?

**Comment by IAmTomahawkx** [comment]
The native mobile apps are currently closed source to prevent people from attempting to use them in their broken/incomplete states (read: filing issues/prs), and to prevent excessive bikeshedding. They will be made public once they are considered usable (eg. once we start doing betas).

**Reply by roke-julian-lockhart** [reply]
@IAmTomahawkx, per [`discussions/804#discussioncomment-10885998`](https://github.com/stoatchat/discussions/discussions/804#discussioncomment-10885998), I presume it's not anymore?

**Reply by IAmTomahawkx** [reply]
That is correct. Both apps are open source and available to use (though still buggy)
